From 6e8aa3749cb7e11e9a59db996f79f036bf7ef263 Mon Sep 17 00:00:00 2001
From: Eugene Syromyatnikov <evgsyr@gmail.com>
Date: Tue, 18 Jan 2022 18:05:19 +0100
Subject: [PATCH 173/174] tests/secontext: eliminate separate secontext_format
 declaration

* tests/secontext.c (secontext_format): Remove declaration, supply
the attributes to the definition.
---
 tests/secontext.c | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/tests/secontext.c b/tests/secontext.c
index 52211ed..ba271c8 100644
--- a/tests/secontext.c
+++ b/tests/secontext.c
@@ -23,10 +23,7 @@
 # define TEST_SECONTEXT
 # include "secontext.h"
 
-static char *
-secontext_format(char *context, const char *fmt)
-	ATTRIBUTE_FORMAT((printf, 2, 0)) ATTRIBUTE_MALLOC;
-
+ATTRIBUTE_FORMAT((printf, 2, 0)) ATTRIBUTE_MALLOC
 static char *
 secontext_format(char *context, const char *fmt)
 {
diff --git a/tests-m32/secontext.c b/tests-m32/secontext.c
index 52211ed..ba271c8 100644
--- a/tests-m32/secontext.c
+++ b/tests-m32/secontext.c
@@ -23,10 +23,7 @@
 # define TEST_SECONTEXT
 # include "secontext.h"
 
-static char *
-secontext_format(char *context, const char *fmt)
-	ATTRIBUTE_FORMAT((printf, 2, 0)) ATTRIBUTE_MALLOC;
-
+ATTRIBUTE_FORMAT((printf, 2, 0)) ATTRIBUTE_MALLOC
 static char *
 secontext_format(char *context, const char *fmt)
 {
diff --git a/tests-mx32/secontext.c b/tests-mx32/secontext.c
index 52211ed..ba271c8 100644
--- a/tests-mx32/secontext.c
+++ b/tests-mx32/secontext.c
@@ -23,10 +23,7 @@
 # define TEST_SECONTEXT
 # include "secontext.h"
 
-static char *
-secontext_format(char *context, const char *fmt)
-	ATTRIBUTE_FORMAT((printf, 2, 0)) ATTRIBUTE_MALLOC;
-
+ATTRIBUTE_FORMAT((printf, 2, 0)) ATTRIBUTE_MALLOC
 static char *
 secontext_format(char *context, const char *fmt)
 {
-- 
2.1.4

